Main/Source/char.cpp

```diff
@@ -2533,7 +2513,14 @@ truth character::TestForPickup(item* ToBeTested) const
 
 void character::AddScoreEntry(cfestring& Description, double Multiplier, truth AddEndLevel) const
 {
-  if(!game::WizardModeIsReallyActive())
+  if(game::WizardModeIsReallyActive())
+    return;
+  if(curseddeveloper::IsCursedDeveloper())
+    return;
+  
+  highscore HScore(game::GetUserDataDir() + HIGH_SCORE_FILENAME);
```

Comment:
I saw the error here:
https://github.com/Attnam/ivan/runs/4004758260?check_suite_focus=true
I think just removing `game::` will solve it like:
`highscore HScore(GetUserDataDir() + HIGH_SCORE_FILENAME);`
but I think there is about 10 other places to do the same fix or it wont fully compile

Reply:
I just created this backup: https://github.com/AquariusPower/ivan/tree/DevTools.TempBackup